recipe/meta.yaml

```diff
@@ -59,16 +63,16 @@ outputs:
     about:
       summary: Library for constructing graphs of media-handling components
       description:
-      doc_source_url: https://cgit.freedesktop.org/gstreamer/gstreamer/tree/docs
+      doc_source_url: https://cgit.freedesktop.org/gstreamer/gstreamer/tree/subprojects/gstreamer/docs
 
   - name: gst-plugins-base
     script: install_base_plugins.sh  # [unix]
     script: install_base_plugins.bat  # [win]
     build:
       ignore_run_exports:
         - gettext                    # [not win]
-        - libpng                     # [linux & ppc64le]
-        - libxcb                     # [linux & ppc64le]
+        - libpng                     # [linux & (ppc64le | s390x)]
+        - libxcb                     # [linux & (ppc64le | s390x)]
```

Comment:
Is there some reason to use `[linux & (ppc64le | s390x)]` over simply `[ppc64le | s390x]`?

Reply:
I'm also curious, but can you also help me understand why we still need to add anything if we are skipping `linux` and `s390x` [here](https://github.com/AnacondaRecipes/gstreamer-feedstock/pull/8/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR23)?

Reply:
Which is better `&`  or `and`? Because only this feedstock uses `&`. Is it supported by conda-build?

Reply:
> only this feedstock uses &

I'll change this, thanks

something like `[linux and (ppc64le | s390x)]` is pretty normal. @ViridianMelody you're right, it's technically redundant, but I suppose we do this for readability. Maybe there's a technical reason I'm not aware of.

@Jrice1317 you're right, this is confusing, I'll remove them. I think we're very unlikely to need to support `s390x` for this package, so it's not worth leaving the info in.